### PR TITLE
Appointments overview page

### PR DESCRIPTION
**Pull request by dimas1976**
This PR is not complete. With this PR I am concerned with the utility method  [formatAppointment](https://github.com/dimas1976/Mentalika/pull/39/files#diff-9af900ab314bf53fa1a8d0a29aab41cb5e21268c2a0172f050ae8801e324a9ae)
I have rewritten it so that I can use the output in different texts. I have rewritten it so that I can use the output in different texts.  For instance [here](https://github.com/dimas1976/Mentalika/pull/39/files#diff-808cd360569fdc178a23be828b31b031ce431d7b84cb3c6c20073c03250d6b88R34) and [here](https://github.com/dimas1976/Mentalika/pull/39/files#diff-f8e5e09d615e49ebb0686c644c7674c2d028b0afbed2944c31c90096cf40a680R21)

Is such a solution suitable for the purpose?

https://mentalika-appointmentso-khjkwp.herokuapp.com/?q=

### CODE REVIEW

File: src/app/pages/AppointmentsOverview/AppointmentsOverview.module.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.page {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  padding: 0 1rem;
+  height: 100vh;
+}
+
+.section {
```
Comment (marwinburesch) [inline src/app/pages/AppointmentsOverview/AppointmentsOverview.module.css:8]: C'mon mate, at least some padding :)
` padding: 1.25rem`

File: src/app/utility/formatAppointment.tsx [hunk 2]
```diff
@@ -19,11 +19,25 @@ function formatMinutesOutput(date: Date): string {
...
-  return appointmentDate;
+  // return appointmentDate;
+
+  return {
+    weekday: getWeekDay(date),
+    date: date.getDate(),
+    month: date.getMonth() + 1,
+    hours: date.getHours(),
+    minutes: formatMinutesOutput(date),
+  };
```
Comment (marwinburesch) [inline src/app/utility/formatAppointment.tsx:42]: No need to be concerned I think this is fine, so does my colleague, Mr. P. 🧐

**Review by marwinburesch — COMMENTED** [review]
👍

File: src/app/utility/formatAppointment.tsx (hunk 2 above)
Comment (dimas1976) [reply src/app/utility/formatAppointment.tsx:42]: Thank you! :)

**Pull request merged**